### PR TITLE
[Enhancement] Logging is added for all materialized view inactive

### LINKED ISSUE #18645: Materialized view inactive is not easy to analyze the cause

### PR DESCRIPTION
**Pull request by Astralidea**
## What type of PR is this：
- [ ] BugFix
- [ ] Feature
- [x] Enhancement
- [ ] Refactor
- [ ] UT
- [ ] Doc
- [ ] Tool

## Which issues of this PR fixes ：

Fixes #18645

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr will affect users' behaviors
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:
- [x] I have checked the version labels which the pr will be auto backported to target branch
  - [x] 3.0
  - [x] 2.5
  - [ ] 2.4
  - [ ] 2.3

### CODE REVIEW

**Comment by Astralidea** [comment]
run starrocks_fe_unittest

**Comment by wanpengfei-git** [comment]
#### [FE PR Coverage Check]

:disappointed: **fail** : 9 / 16 (56.25%)




#### file detail

|   |path|covered_line|new_line|coverage|not_covered_line_detail|
|----|----|----|----|----|----|
|:large_blue_circle:|com/starrocks/server/GlobalStateMgr.java|0|6|00.00%|[1417, 1418, 1423, 1424, 1425, 1426]|
|:large_blue_circle:|com/starrocks/server/LocalMetastore.java|3|4|75.00%|[3864]|
|:large_blue_circle:|com/starrocks/catalog/OlapTable.java|2|2|100.00%|[]|
|:large_blue_circle:|com/starrocks/alter/SchemaChangeJobV2.java|4|4|100.00%|[]|

File: fe/fe-core/src/test/java/com/starrocks/catalog/MaterializedViewTest.java [hunk 1]
```diff
@@ -541,40 +541,6 @@ public void testMvAfterBaseTableRename() throws Exception {
         Assert.assertFalse(mv.isActive());
     }
 
-    @Test
-    public void testMvAfterBaseTablePartitionRename() throws Exception {
```
Comment (LiShuMing) [inline fe/fe-core/src/test/java/com/starrocks/catalog/MaterializedViewTest.java:545]: Why remove this?

**Review by ABingHuang — APPROVED** [review]
LGTM

**Review by murphyatwork — APPROVED** [review]

**Comment by wanpengfei-git** [comment]
run starrocks_admit_test

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Comment by Astralidea** [comment]
run starrocks_admit_test

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-3.0

**Comment by wanpengfei-git** [comment]
@Mergifyio backport branch-2.5

**Pull request merged**